contracts/utils/Arrays.sol

```diff
@@ -75,6 +127,43 @@ library Arrays {
         }
     }
 
+    /// @dev Comparator for sorting arrays in increassing order.
+    function _defaultComp(bytes32 a, bytes32 b) private pure returns (bool) {
+        return uint256(a) < uint256(b);
+    }
```

Comment:
I wonder if the default comp function should have `uint256, uint256` arguments. It's weird that we convert all comparison functions to `bytes32` and at the end we compare `uint256`.

The benefit is more semantical, I think it's easier to follow that a function compares numbers and not bytes.

Not sure if it's even possible though, there may be a casting limitation I'm missing

Reply:
I wanted the default to be about bytes32[], because I feel that is the most "fundamental" type. This is possibly going to change if we do #4883 and operate on pointers directly.

Reply:
I changes the base type (returned by `_mload`, and used for the comparator) to uint256

Reply:
resolved: it turns out `<` is available for `bytes32`